lib/ros/be/application/platform/templates/skaffold/service.yml.erb

```diff
@@ -37,6 +37,8 @@ profiles:
             image.tag: "{{.IMAGE_TAG}}"
             image.repository: "{{.SKAFFOLD_DEFAULT_REPO}}/<%= @service.name %>"
           overrides:
+            podDisruptionBudget: |
+              minAvailable: 1
```

Comment:
Why here the value for podDisruptionBudget is a literal block instead of an object?

Reply:
And I think the section in the helm-chart should be modified to use function `toYaml`:

https://github.com/rails-on-services/helm-charts/blob/master/charts/service/templates/pdb.yaml#L11

Compared with https://github.com/rails-on-services/helm-charts/blob/master/charts/service/templates/deployment.yaml#L134